conjur/init_logic.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+
+"""
+InitLogic module
+
+This module is the business logic for writing configuration information
+to the user's machine as well as fetching certificates from Conjur
+"""
+
+# Builtins
+import logging
+import sys
+import os.path
+
+# Third party
+import socket
+import yaml
+from OpenSSL import SSL
+from OpenSSL.crypto import FILETYPE_PEM
+
+# pylint: disable=raise-missing-from
+class InitLogic:
+    """
+    InitLogic
+
+    This class holds the business logic for population the
+    conjurrc configuration details needed to connect with Conjur
+    """
+    DEFAULT_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.conjurrc'))
+    DEFAULT_PROMPT_RESPONSE=True
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def connect_to_socket(hostname, port):
+        """
+        Method for opening a socket with the Conjur server
+        """
+        try:
+            if port is None:
```

Comment:
`port = 443 if port is None else port`

Reply:
got a comment from srdjan to break it up for readability purposes. Made the 443 a constant